editor/editor_settings.cpp

```diff
@@ -363,137 +371,105 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 			best = "en";
 		}
 
-		_initial_set("interface/editor/editor_language", best);
+		EDITOR_SETTING_USAGE(STRING, ENUM, "interface/editor/editor_language", best, lang_hint, PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_RESTART_IF_CHANGED);
 		set_restart_if_changed("interface/editor/editor_language", true);
```

Comment:
Does anyone know why we have both `set_restart_if_changed` and `PROPERTY_USAGE_RESTART_IF_CHANGED`?

Reply:
I don't know why but that does seem redundant. I'm not sure both are 100% equivalent though, if you look at `set_restart_if_changed` it sets a `restart_if_changed` struct member which is then used later on to set the usage flag.

Since here the usage flag is set manually, the method call can likely be removed. But that mean that `restart_if_changed` won't be set to `true` in the struct.